Makefile

```diff
@@ -45,6 +45,11 @@ API_OPTIONS?=
 # See http://stackoverflow.com/questions/18136918/how-to-get-current-relative-directory-of-your-makefile
 MAKEDIR:=$(strip $(shell dirname "$(realpath $(lastword $(MAKEFILE_LIST)))"))
 
+# Unexport environment variables that can affect tests and are not used in builds
```

Comment:
@rifelpet Nit: could we sort these in alphabetical order? It will make future management of the list less error-prone.

Reply:
Yeah, I have them categorized by kops-specific, aws-specific, and other cloud providers, but ill switch it to entirely alphabetical